llvm/lib/Target/AArch64/AArch64AsmPrinter.cpp

```diff
@@ -160,8 +160,7 @@ class AArch64AsmPrinter : public AsmPrinter {
 
   /// tblgen'erated driver function for lowering simple MI->MC
   /// pseudo instructions.
-  bool emitPseudoExpansionLowering(MCStreamer &OutStreamer,
```

Comment:
The old function name is better? This new name has no connection to the tablegen class

Reply:
It no longer emits, so I had to rename it.

I chose that name because it effectively lowers MachineInstr to MCInst, much like non-generated versions,
e.g. `LowerARMMachineInstrToMCInst` or `RISCVAsmPrinter::lowerToMCInst` or `AMDGPUMCInstLower::lower`
(and `Impl` suffix so that it doesn't conflict with non-autogenerated versions, just in case). The set of arguments is also the same in most cases.

Would something of these be less confusing to you, or can you suggest better alternative without `emit` prefix?
- expandPseudoMachineInstrToMCInst
- expandPseudoInstrToMCInst
- expandPseudoMachineInstr
- expandPseudoInstr
- s/expand/lower one of these

Reply:
@arsenm Is the new name better? It directly refers to the tablegen class.